alcs-frontend/src/app/services/application/application.dto.ts

```diff
@@ -21,6 +22,7 @@ export interface ApplicationDto {
   status: string;
   summary?: string;
   type: string;
+  // cardUuid: string;
```

Comment:
Commented out?

Reply:
yes, there is "card: CardDto". I'll remove commented property